cmd/postcli/README.md

```diff
@@ -158,3 +158,19 @@ in each `post_metadata.json` of every subset. Given two files:
 
 The nonce in the second file is the global minimum since its value is smaller than the first one. The operator is **required** to find the
 smallest VRF nonce by hand and ensure that its index and value are in the `postdata_metadata.json` of the merged directory on the target machine.
+
+
+## Verifying initialized POS data
+
+The `postcli` allows verifying an already initialized POS data. Verification randomly picks a small fraction of labels from every file and compares them to labels generated with the scrypt-jane algorithm executed on CPU. Please note that generating labels on CPU is slow, compared to GPU. Hence it is not possible to verify all the data (it would essentially mean re-initialization on CPU). The assumption is that if the GPU process breaks, it starts to output large chunks of invalid labels. This method will not detect a single invalid byte/label.
```

Comment:
```suggestion
The `postcli` allows verifying an already initialized POS data. Verification samples a small fraction of labels from every file and compares them to labels generated with the same algorithm executed on CPU. Please note that generating labels on CPU is slow compared to GPU. Hence it is not possible to verify all the data (it would essentially mean re-initialization on CPU). If the GPU failed during initialization, the created PoST data will contain some or all invalid labels after that point. This method will only sample the PoST and might not detect a small amount of corrupted data.
```